### DISCUSSION TITLE
Does MediaLibrary work with Livewire V3

### DISCUSSION THREAD

**Discussion by dellow** [spatie/laravel-medialibrary#3343] [Q&A]
As title really. Does this work with Livewire V3, if not are there plans to support?

**Comment by freekmurze** [comment]
This package is not Livewire specific. [Our pro package](https://medialibrary.pro) is, and we're currently updating it for Livewire v3. ETA: a few weeks.

**Reply by dellow** [reply]
Perfect, thanks.
